Makefile

```diff
@@ -400,10 +408,23 @@ root-coverage: ## generate coverage profiles for unit tests that require root
 		fi; \
 	done )
 
-vendor: ## vendor
+vendor: ## ensure all the go.mod/go.sum files are up-to-date including vendor/ directory
 	@echo "$(WHALE) $@"
-	@$(GO) mod tidy
-	@$(GO) mod vendor
+	@go mod tidy
+	@go mod vendor
+	@(cd ${ROOTDIR}/api && go mod tidy)
+	@(cd ${ROOTDIR}/integration/client && go mod tidy)
+
+verify-vendor: ## verify if all the go.mod/go.sum files are up-to-date
```

Comment:
I like how the cri-o guys are verifying vendor by using git status
https://github.com/cri-o/cri-o/blob/master/Makefile#L278-L279
https://github.com/cri-o/cri-o/blob/master/hack/tree_status.sh#L4

Reply:
The issue with that approach @mikebrow is that it blows away things you have in your work area

Reply:
good point..